src/components/Common/QuickIngestModal.tsx

```
@@ -177,16 +321,110 @@ export const QuickIngestModal: React.FC<Props> = ({ open, onClose }) => {
+      )
+      return
+    }
     const valid = rows.filter((r) => r.url.trim().length > 0)
     if (valid.length === 0 && localFiles.length === 0) {
       messageApi.error('Please add at least one URL or file')
       return
     }
     const total = valid.length + localFiles.length
     setTotalPlanned(total)
+
+  const autoProcessedRef = React.useRef(false)
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)